### PR TITLE
Update README to be more explicit[Closes #45]

### LINKED ISSUE #45: Update README
As a developer I want clear instructions for setting up my local environment

### PR DESCRIPTION
**Pull request by nodatall**
Püll request'é for ¥ou, dearest JROß

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,56 @@
...
+```
+$ brew install yarn
+```
+### Install Mongo
+```
+$ brew install mongodb
+$ brew services start mongodb
+```
 
+###### Create db directory
```
Comment (jrobcodes) [inline README.md:24]: I don't think these steps are necessary if mongo is installed via homebrew? (This one through `Run mongo daemon`.)

**Review by jrobcodes — CHANGES_REQUESTED** [review]
@nodatall @Moniarchy comments inline

File: README.md (hunk 1 above)
Comment (EthanJStark) [reply README.md:24]: @jrob8577 As a guinea pig, I hearby testify that the mkdir and chown commands were, in fact required. Without them,  it shut down with error 100

Comment (Moniarchy) [reply README.md:24]: Thanks @EthanJStark !

**Review by jrobcodes — APPROVED** [review]

**Pull request merged**